File: react-front/SlayTheSpire2React-font/src/types/relic.ts
export interface Relic {
  id: number
  name: string
  urlImage: string
  rarity: string
  description: string
  characters: string
}

export interface CreateRelicPayload {
  name: string
  urlImage: string
  rarity: string
  description: string
  characters: string
}

File: Net23Online/SlayTheSpire2RelicsMinimalApi/DbStuff/Relic.cs
namespace SlayTheSpire2RelicsMinimalApi.DbStuff
{
    public class Relic
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string UrlImage { get; set; }
        public string Rarity { get; set; }
        public string Description { get; set; } = "";
        public string Characters { get; set; } = "All";
    }
}
